controllers/search/search.go

```diff
@@ -64,6 +64,10 @@ func SearchHandler(c *gin.Context) {
 	if c.Query("order") == "true" {
 		searchForm.SortOrder = true
 	}
+	searchForm.SortType, err = strconv.Atoi(c.Query("sort"))
```

Comment:
alright but i already tried making the changes you requested and i miserably failed at it so i can't really do it anyway